### PR TITLE
Add sample: 201-front-door-standard-premium-geo-filtering

### PR DESCRIPTION
**Pull request by johndowns**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Add sample: 201-front-door-standard-premium-geo-filtering

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by azure-quickstarts** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

File: 201-front-door-standard-premium-geo-filtering/main.bicep [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+param originHostName string
+
+@description('The name of the SKU to use when creating the Front Door profile.')
+@allowed([
+  'Standard_AzureFrontDoor'
+  'Premium_AzureFrontDoor'
+])
+param frontDoorSkuName string = 'Standard_AzureFrontDoor'
+
+@description('The mode that the WAF should be deployed using. In \'Prevention\' mode, the WAF will block requests it detects as malicious. In \'Detection\' mode, the WAF will not block requests and will simply log the request.')
```
Comment (bmoore-msft) [inline 201-front-door-standard-premium-geo-filtering/main.bicep:14]: Can we just use double quotes inside the string to remove the escaping?

File: 201-front-door-standard-premium-geo-filtering/main.bicep [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+@allowed([
+  'Detection'
+  'Prevention'
+])
+param wafMode string = 'Prevention'
+
+@description('The list of country codes that are allowed to access the Front Door endpoint.')
```
Comment (bmoore-msft) [inline 201-front-door-standard-premium-geo-filtering/main.bicep:21]: Are allowed values defined by the RP or the developer?  If the latter where do I get the list of codes?

File: 201-front-door-standard-premium-geo-filtering/main.bicep (hunk 1 above)
Comment (johndowns) [reply 201-front-door-standard-premium-geo-filtering/main.bicep:14]: Given these are rendered using Markdown (as far as I know at least?) in the portal UI and potentially other tooling, I felt that using the backticks was the right thing to do for literal values. If you'd like me to replace them with double quotes I can though.

File: 201-front-door-standard-premium-geo-filtering/main.bicep (hunk 2 above)
Comment (johndowns) [reply 201-front-door-standard-premium-geo-filtering/main.bicep]: Updated to clarify this in the description as well as in the README.md.

**Comment by azure-quickstarts** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Comment by azure-quickstarts** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

File: 201-front-door-standard-premium-geo-filtering/main.bicep (hunk 1 above)
Comment (johndowns) [reply 201-front-door-standard-premium-geo-filtering/main.bicep]: Updated.

**Comment by azure-quickstarts** [comment]
@johndowns - check this PR for updates that may be needed to documentation that references this sample.  [this is an automated message]

**Pull request merged**